benchmarks_gui/CMakeLists.txt

```diff
@@ -12,8 +12,18 @@ find_package(PkgConfig REQUIRED)
 pkg_check_modules(OGRE OGRE)
 link_directories(${OGRE_LIBRARY_DIRS})
 
-find_package(Qt4 REQUIRED)
-include(${QT_USE_FILE})
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+option(UseQt5 "UseQt5" OFF)
```

Comment:
IMHO, the rviz plugin should use exactly the same Qt version as rviz.
Please use a similar pattern as in https://github.com/ros-visualization/visualization_tutorials/pull/33/commits/703931401d05948dd4e699fbfd75201c13af559b